src/main/java/appeng/core/settings/TickRates.java

```diff
@@ -37,6 +37,8 @@
 
 	Inscriber( 1, 1 ),
 
+	Charger( 10, 120 ),
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00115)